node/src/Transaction.ts

```diff
@@ -1035,6 +1038,29 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
         );
     }
 
+    /**
+     * Computes the intersection of sorted sets given by the specified `keys` and stores the result in `destination`.
+     * If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.
+     * When in cluster mode, `destination` and all keys in `keys` must map to the same hash slot.
```

Comment:
We don't add this for transaction.
```suggestion
```

Reply:
so to remove it from transaction.py as well?

Reply:
why we are not adding it in transaction?

Reply:
I dont think we need to remove it. it is relevant for transaction as well.
If you'll find a reason please update but currently i see it as a mistake.